internal/services/servicebus/servicebus_queue_data_source_test.go

```diff
@@ -53,9 +53,9 @@ func TestAccDataSourceServiceBusQueue_withNamespaceName(t *testing.T) {
 				check.That(data.ResourceName).Key("dead_lettering_on_message_expiration").Exists(),
 				check.That(data.ResourceName).Key("default_message_ttl").Exists(),
 				check.That(data.ResourceName).Key("duplicate_detection_history_time_window").Exists(),
-				check.That(data.ResourceName).Key("enable_batched_operations").Exists(),
-				check.That(data.ResourceName).Key("enable_express").Exists(),
-				check.That(data.ResourceName).Key("enable_partitioning").Exists(),
+				check.That(data.ResourceName).Key("batched_operations_enabled").Exists(),
+				check.That(data.ResourceName).Key("express_enabled").Exists(),
+				check.That(data.ResourceName).Key("partitioning_enabled").Exists(),
```

Comment:
is this test needed anymore, now that `namespace_name` is deprecated?

Reply:
It's not but we should keep it around to confirm that this functionality still works until we remove it in 5.0. So we can add a check to skip this if we're in 5.0 beta and then we can remove it later